js/msp.js

```diff
@@ -1510,6 +1506,9 @@ MSP.crunch = function (code) {
             for (var i = 0; i<CONFIG.name.length; i++) {
```

Comment:
What does the FC do if for some reason `CONFIG.name` is much larger than 16 here? I would probably add a check for maximum allowable length here, to avoid the risk of sending something that the FC can't handle.